src/main/java/io/reactivex/exceptions/OnCompleteFailedException.java

```diff
@@ -18,7 +18,11 @@
     private static final long serialVersionUID = -6179993283427447098L;
 
     public OnCompleteFailedException(Throwable cause) {
-        super(cause);
+        super(cause != null ? cause : new NullPointerException());
```

Comment:
This has been discussed back then: instead of crashing the code further, it sets an NPE with the current stack trace to give something to the receiver.